articles/guides/login/migration-sso.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Migration in Embedded Login Scenarios with SSO
+description: Learn how to migrate from old versions of Lock/Auth0.js when your application uses embedded login and requires SSO.
+---
+
+# Migration in Embedded Login Scenarios with SSO
+
+Migration from legacy versions of Lock and Auth0.js to the most current versions is required. However, for scenarios in which Single Sign On (SSO) is required, the migration path can be more complex. In most cases where SSO is required, Auth0 recommends use of [Universal Login](/hosted-pages/login). 
+
+## Single Page Apps
+
+Single Page Applications (SPAs) with embedded login can only achieve SSO if they are on the same top-level domain. If SPAs with embedded login which are on different domains require SSO, Auth0's recommendation is that they [migrate to Universal Login](/guides/login/migration-embedded-universal).
+
```

Comment:
If SPAs with embedded login which are on different domains require SSO, the websites will need to [migrate to Universal Login](/guides/login/migration-embedded-universal).